src/backend/docker-compose.yml

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Let's not make whitespace changes 

Even for future pull requests as well

Reply:
ok sir